### ISSUE TITLE
Can't edit own user details in NC 18.0.1

### ISSUE THREAD

**Issue by arnowelzel** [nextcloud/server#19780]
## Steps to reproduce
1. Set up NC 17 and log in as administrator
2. Assign many groups to the administrator, so the group list does not fit the user row in the UI after upgrading to NC 18.01
3. Upgrade to NC 18.0.1
4. Login as administrator again
5. Open the user administration

### Expected behaviour
Every user row is fully visible and can be edited.

### Actual behaviour
The whole row gets enlarged so it will not be displayed properly again. Also the "edit" icon is not visible any longer, so the user can't be edited - in this case I am logged in as "Arno Welzel" and I can't edit any details of "Arno Welzel" which is in fact myself:

![image](https://user-images.githubusercontent.com/6613614/75898479-668f4180-5e3a-11ea-8118-0f0d2e90a612.png)

The UI issue with overlapping rows is also described in #18968, but I don't understand, why there is no "edit" icon any longer for the user with which I am logged in.

In NC 17 this was no problem at all - every user including the one with which I am currently logged in, can be edited and the rows would enlarge if needed to display all details of a user.

BTW: I also don't understand why the quota field only shows the relative usage and the value itself got completely removed - at least the numerical values should be visible here as well, because it makes a big difference if half of 1 GB is used or half of 1000 GB! In previous versions it was much better: an input field with the current quota limit and below that the line with the relative usage.

### Server configuration

**Operating system:**
Ubuntu Linux 16.04

**Web server:**
Apache 2.4

**Database:**
MariaDB 10.3

**PHP version:**
7.3

**Nextcloud version:**
18.0.1

**Updated from an older Nextcloud/ownCloud or fresh install:**
Update

**Where did you install Nextcloud from:**
nextcloud.com

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```Enabled:
  - accessibility: 1.4.0
  - activity: 2.11.0
  - apporder: 0.9.0
  - audioplayer: 2.9.0
  - bruteforcesettings: 1.5.0
  - calendar: 2.0.2
  - cloud_federation_api: 1.1.0
  - comments: 1.8.0
  - contacts: 3.2.0
  - dav: 1.14.0
  - deck: 0.8.0
  - drawio: 0.9.5
  - federatedfilesharing: 1.8.0
  - federation: 1.8.0
  - files: 1.13.1
  - files_downloadactivity: 1.7.0
  - files_pdfviewer: 1.7.0
  - files_rightclick: 0.15.2
  - files_sharing: 1.10.1
  - files_trashbin: 1.8.0
  - files_versions: 1.11.0
  - files_videoplayer: 1.7.0
  - firstrunwizard: 2.7.0
  - gpxpod: 4.1.1
  - keeweb: 0.6.2
  - logreader: 2.3.0
  - lookup_server_connector: 1.6.0
  - news: 14.1.3
  - nextcloud_announcements: 1.7.0
  - notes: 3.1.5
  - notifications: 2.6.0
  - oauth2: 1.6.0
  - onlyoffice: 4.1.4
  - password_policy: 1.8.0
  - photos: 1.0.0
  - privacy: 1.2.0
  - provisioning_api: 1.8.0
  - serverinfo: 1.8.0
  - settings: 1.0.0
  - sharebymail: 1.8.0
  - socialsharing_email: 2.0.0
  - spreed: 8.0.5
  - support: 1.1.0
  - survey_client: 1.6.0
  - systemtags: 1.8.0
  - tasks: 0.11.3
  - text: 2.0.0
  - theming: 1.9.0
  - theming_customcss: 1.5.0
  - twofactor_backupcodes: 1.7.0
  - updatenotification: 1.8.0
  - viewer: 1.2.0
  - workflowengine: 2.0.0
Disabled:
  - admin_audit
  - encryption
  - files_external
  - quota_warning
  - recommendations
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "nextcloud.0x0c.de"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "skeletondirectory": "",
        "overwrite.cli.url": "https:\/\/nextcloud.0x0c.de",
        "dbtype": "mysql",
        "version": "18.0.1.3",
        "installed": true,
        "htaccess.RewriteBase": "\/",
        "maintenance": false,
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "knowledgebaseenabled": false,
        "theme": "",
        "loglevel": 2,
        "log_authfailip": true,
        "mysql.utf8mb4": true,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "sendmail",
        "mail_smtpauthtype": "LOGIN",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "updater.release.channel": "stable",
        "trashbin_retention_obligation": "auto, 30",
        "app_install_overwrite": [
            "apporder",
            "deck"
        ],
        "has_rebuilt_cache": true
    }
}
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** n/a

### Client configuration
**Browser:**
Firefox 73.0.1

**Operating system:**
Ubuntu Linux 18.04

#### Browser log
<details>
<summary>Browser log</summary>

```
JQMIGRATE: Migrate is installed, version 1.4.1 jquery-migrate.min.js:2:551
This page uses the non standard property “zoom”. Consider using calc() in the relevant property values, or using “transform” along with “transform-origin: 0 0”. addStylesClient.js:220:17
Proxying an event bus of version 1.1.2 with 0.2.1 ProxyBus.js:65:14
Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:66:15
New search handler registered search.js:67:12
```
</details>

**Comment by kesselb** [comment]
Duplicate of #18530, Duplicate of #18964, Duplicate of #19004, Duplicate of #19072

**Comment by kesselb** [comment]
Duplicate of #18964

**Comment by kesselb** [comment]
Duplicate of #19004

**Comment by kesselb** [comment]
Duplicate of #19072

**Comment by kesselb** [comment]
OK.  Something weird happened as I used `Duplicate of #a, #b, #c` ;) Fix is scheduled for 18.0.2.

**Comment by arnowelzel** [comment]
Thank you!
